test/test.js

```diff
@@ -0,0 +1,10 @@
+/* global describe it */
+/* eslint-disable no-unused-expressions */
```

Comment:
Let's have all this as global config for now

Reply:
... maybe not...
The [eslint-disable can stay](https://stackoverflow.com/questions/37558795/nice-way-to-get-rid-of-no-unused-expressions-linter-error-with-chai), but the globals can be moved as `env: mocha` to the general config